ArduSub/Parameters.h

```diff
@@ -418,9 +418,10 @@ extern const AP_Param::Info        var_info[];
 // Sub-specific default parameters
 static const struct AP_Param::defaults_table_struct defaults_table[] = {
     { "BRD_SAFETY_DEFLT",    0 },
-    { "ARMING_CHECK",        uint32_t(AP_Arming::Check::RC) |
-                             uint32_t(AP_Arming::Check::VOLTAGE) |
-                             uint32_t(AP_Arming::Check::BATTERY)},
+    { "ARMING_SKIPCHK",      (~(uint32_t(AP_Arming::Check::RC) |
+                                uint32_t(AP_Arming::Check::VOLTAGE) |
+                                uint32_t(AP_Arming::Check::BATTERY))
+                               ) & ((1U<<24)-1)}, // keep within float range but disable future checks
```

Comment:
should be CHECK_LAST-1 ?

Reply:
No, we want future checks to be disabled if the user touches the parameter. Modulo the whole float storage stuff.